src/components/types/enums.js

```diff
@@ -333,6 +333,14 @@ export const TextAlign = {
   right: 'right'
 };
 
+export const TextEntryMode = {
+  email: 'email',
+  none: 'none',
```

Comment:
Does `none` mean that the keyboard is not presented and the Text Field is treated like a Label?

Reply:
```
case none // Virtual Keyboard does not display
```

Reply:
Something like that - I don't remember how it behaves on Lumin, but on iOS we don't show the keyboard for the 'none' text entry mode.